test_add_remove_ldap_role_permission.py

```diff
@@ -0,0 +1,91 @@
+                                   is_login=True)
+        if self.params.get('prepare_saslauthd'):
+            self.add_user_in_ldap(username=new_test_user)
+        with pytest.raises(Unauthorized, match="has no CREATE permission"):
+            with self.db_cluster.cql_connection_patient(node=node, user=new_test_user, password=LDAP_PASSWORD) as session:
+                session.execute(
+                    f""" CREATE KEYSPACE IF NOT EXISTS {ldap_keyspace} WITH replication = {{'class': 'SimpleStrategy',
+                    'replication_factor': 1}} """)
+
+        self.log.debug("Create a new super-user role in Scylla")
+        self.create_role_in_scylla(node=node, role_name=superuser_role, is_superuser=True,
+                                   is_login=False)
+        self.db_cluster.wait_for_schema_agreement()
+        self.log.debug("Create a new super-user role in Ldap, associated with the new user")
+
+        self.create_role_in_ldap(ldap_role_name=superuser_role, unique_members=[new_test_user, LDAP_USERS[1]])
+
+        self.wait_for_user_roles_update(are_roles_expected=True, username=new_test_user)
+        self.log.debug("Create keyspace and table where authorized")
+        self.wait_verify_user_permissions(username=new_test_user, keyspace=ldap_keyspace)
+
+        # Run a stress while new user permissions are removed
+        new_test_user_stress = f"cassandra-stress write no-warmup cl=QUORUM duration=10m -schema 'keyspace={ldap_keyspace} " \
+                               f" replication(factor=3)' -mode cql3 native user={new_test_user} password={LDAP_PASSWORD}" \
+                               " -rate threads=2 -pop seq=1..1002003 -log interval=5"
+        stress_queue.append(self.run_stress_thread(stress_cmd=new_test_user_stress, round_robin=True))
+        self.log.debug("Remove authorization and verify unauthorized user")
+        self.modify_ldap_role_delete_member(ldap_role_name=superuser_role, member_name=new_test_user)
+        self.wait_for_user_roles_update(are_roles_expected=False, username=new_test_user)
+        self.wait_verify_user_no_permissions(username=new_test_user)
+        for stress in stress_queue:
+            self.verify_stress_thread(cs_thread_pool=stress)
+
+        # Clean-up resources
+        self.delete_ldap_role(ldap_role_name=superuser_role)
+
+        with self.db_cluster.cql_connection_patient(node=node, user=LDAP_USERS[0], password=LDAP_PASSWORD) as session:
+
+            session.execute(f""" DROP TABLE IF EXISTS {ldap_keyspace}.info """)
+            session.execute(f""" DROP TABLE IF EXISTS {ldap_keyspace}.standard1 """)
+            session.execute(f""" DROP KEYSPACE IF EXISTS {ldap_keyspace} """)
+            session.execute(""" DROP KEYSPACE IF EXISTS test_no_permission """)
+            session.execute(""" DROP KEYSPACE IF EXISTS test_permission_ks """)
+            session.execute(f"DROP ROLE IF EXISTS {superuser_role}")
```

Comment:
shouldn't we try also to run these `drop` commands for a user that has no permissions too?